dev-util/edi/edi-0.7.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit llvm meson xdg-utils
+
+DESCRIPTION="An EFL-based IDE"
+HOMEPAGE="https://www.enlightenment.org/about-edi"
+SRC_URI="https://github.com/Enlightenment/edi/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 LGPL-2.1-with-linking-exception"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="clang"
+
+RDEPEND="
+	>=dev-libs/efl-1.21.1[eet,X]
+	clang? ( sys-devel/clang:${LLVM_SLOT}= )
+"
+DEPEND="
+	${RDEPEND}
+	dev-libs/check
+"
+
```

Comment:
You're missing `llvm_check_deps()` to ensure that your selected slot actually includes clang.